examples/resources/grafana_cloud_private_datasource_connect/resource.tf

```
@@ -0,0 +1,17 @@
+data "grafana_cloud_stack" "current" {
+  stackID = "<your stack ID>"
+}
+
+resource "grafana_cloud_private_datasource_connect" "test" {
```

This is the PDC network, I'd like to be explicit with naming
```suggestion
resource "grafana_cloud_private_datasource_connect_network" "test" {
```

---

Done!